pkgs/games/fish-fillets-ng/default.nix

```
@@ -10,18 +11,34 @@ stdenv.mkDerivation rec {
     url = "mirror://sourceforge/fillets/fillets-ng-data-${version}.tar.gz";
     sha256 = "169p0yqh2gxvhdilvjc2ld8aap7lv2nhkhkg4i1hlmgc6pxpkjgh";
   };
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [SDL lua5_1 SDL_mixer SDL_image SDL_ttf];
-  postInstall=''
+
+  nativeBuildInputs = [ pkgconfig copyDesktopItems ];
+  buildInputs = [ SDL lua5_1 SDL_mixer SDL_image SDL_ttf ];
+
+  desktopItems = [ (makeDesktopItem {
+    name = "fish-fillets-ng";
+    exec = "fillets";
+    icon = "fish-fillets-ng";
+    desktopName = "Fish Fillets";
+    comment     = "Puzzle game about witty fish saving the world sokoban-style";
+    categories  = "Game;LogicGame;";
+  }) ];
+
+  postInstall = ''
+    # Install game data
     mkdir -p "$out/share/games/fillets-ng/"
     tar -xf ${data} -C "$out/share/games/fillets-ng/" --strip-components=1
+
+    # Install game icon
+    install -Dm644 ${./icon.xpm} $out/share/pixmaps/fish-fillets-ng.xpm
```

```suggestion
    mkdir -p $out/share/games/fillets-ng/
    tar -xf ${data} -C $out/share/games/fillets-ng/ --strip-components=1
    install -Dm644 ${./icon.xpm} $out/share/pixmaps/fish-fillets-ng.xpm
```

out can't contain spaces.